### DISCUSSION TITLE
Disable upvotes

### DISCUSSION THREAD

**Discussion by touhf** [Docile-Alligator/Infinity-For-Reddit#1019] [Ideas]
Option to completely disable upvotes from app so that they would not be showed anywhere (in comments, posts, profile)

**Comment by Wladefant** [comment]
So specifically a toggle in settings>interface>post details 'Hide Vote Option'
And the same in ...>comment 
Acceptable?

**Reply by touhf** [reply]
Yeah,
and also to add 'Hide Number of Votes' in ..>comment settings like it's made in ..>post
and option to hide karma score in my profile and menu.
That would be awesome

**Reply by Wladefant** [reply]
New feature requests please in a separate post
